src/lib/packages.ts

```diff
@@ -284,7 +283,7 @@ const actOnTailwindGenerator = (attribute: string) => {
       codeToCopy = ``;
       break;
     case 'border-radius':
-      codeToCopy = ``;
+      codeToCopy = `<div class="rounded-sm"></div>`;
```

Comment:
This is hard coded and not what this project is about, please look at what this person did for this PR, this is what is expected https://github.com/Dun-sin/Code-Magic/pull/421